### PR TITLE
Conti/add aws kinesis tests

### PR DESCRIPTION
**Pull request by wconti27**
## Motivation

More testing

## Changes

Adds Kinesis cross integration tests, and implements endpoint for dd-trace-py

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] [Relevant label](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md) (`run-parametric-scenario`, `run-profiling-scenario`...) are presents
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
    * [ ] To R&P team: locally build and push the image to hub.docker.com 
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
    * [ ] Once merged, add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

File: tests/integrations/crossed_integrations/test_kinesis.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+#     @missing_feature(library="ruby", reason="Expected to fail, Ruby does not propagate context")
+#     def test_produce_trace_equality(self):
+#         super().test_produce_trace_equality()
+
+#     @missing_feature(library="golang", reason="Expected to fail, Golang does not propagate context")
+#     @missing_feature(library="ruby", reason="Expected to fail, Ruby does not propagate context")
+#     @missing_feature(library="python", reason="Expected to fail, Python does not propagate context")
+#     @missing_feature(library="nodejs", reason="Expected to fail, Nodejs does not propagate context")
+#     def test_consume_trace_equality(self):
+#         super().test_consume_trace_equality()
```
Comment (robertomonteromiguel) [inline tests/integrations/crossed_integrations/test_kinesis.py:232]: I advocate deleting these lines

**Review by robertomonteromiguel — APPROVED** [review]
Only a minor comment

File: tests/integrations/crossed_integrations/test_kinesis.py (hunk 1 above)
Comment (wconti27) [reply tests/integrations/crossed_integrations/test_kinesis.py:232]: done.

File: utils/build/docker/python/flask-poc.Dockerfile [hunk 2]
```diff
@@ -19,6 +19,8 @@ ENV _DD_APPSEC_DEDUPLICATION_ENABLED=false
 ENV DD_BOTOCORE_PROPAGATION_ENABLED=true
 ENV DD_KAFKA_PROPAGATION_ENABLED=true
 
+ENV LOG_LEVEL='DEBUG'
```
Comment (cbeauchesne) [inline utils/build/docker/python/flask-poc.Dockerfile:22]: This was causing lot of flakiness on the python tracer, we removed this line a while ago.

**Pull request merged**